### PR TITLE
create dataset with email options

### PR DESCRIPTION
**Pull request by jacobmanderson**
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] Jira story/task is put into PR title
- [ ] Code has been self-reviewed
- [ ] Hard-to-understand areas have been commented
- [ ] Documentation has been updated
- [ ] My changes generate no new warnings
- [ ] Unit tests have been added for base functionality and known edge cases
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: indico/types/dataset.py [hunk 1]
```diff
@@ -115,20 +124,35 @@ class ReadApiOcrOptionsInput(BaseType):
...
 
-class OcrOptionsInput():
+
+class IncludeSections(BaseType):
+    header: Optional[bool]
+    body: Optional[bool]
+    attachments: Optional[bool]
+
+
+class EmailOptions(BaseType):
```
Comment (nicholas-lockhart) [inline indico/types/dataset.py:145]: This would benefit with an Args comment. Unpack is a bit ambiguous, so it would be good to have a reminder later what the heck we were thinking about.

**Review by sihrc — APPROVED** [review]

**Review by nicholas-lockhart — APPROVED** [review]
